.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This and some other files look eclipse specific and we can probably put them in the gitignore. not a big deal though.